ij/src/main/kotlin/org/jetbrains/academy/test/system/ij/formatting/BaseIjTestClass.kt

```diff
@@ -0,0 +1,7 @@
+package org.jetbrains.academy.test.system.ij.formatting
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+
+open class BaseIjTestClass : BasePlatformTestCase() {
```

Comment:
Is it empty on purpose?

Reply:
This class is needed to avoid dragging dependency into the course. Tests in which psi is needed can be added to this class in the future.